etc/releasing.md

```diff
@@ -85,237 +165,298 @@ Review the contents of the new Augmented SBOM and ensure any new or known vulner
+The following secrets are required by this script:
 
-- A GitHub token. Go to the GitHub settings page
-  [Personal Access Tokens](https://github.com/settings/tokens) and create a
-  token.  Save the token secret to `~/.secrets/github_token.txt`.
-- Jira OAuth credentials. Save it to `~/.secrets/jira_creds.txt`.
-- Artifactory credentials. Save these to `~/.secrets/artifactory-creds.txt`:
-  ```bash
-  ARTIFACTORY_USER=<username>
-  ARTIFACTORY_PASSWORD=<password>
-  ```
-- Garasign credentials. Save these to `~/.secrets/garasign-creds.txt`:
-  ```bash
-  GRS_CONFIG_USER1_USERNAME=<username>
-  GRS_CONFIG_USER1_PASSWORD=<password>
-  ```
-- Silk credentials. Save these to `~/.secrets/silk-creds.txt`:
-  ```bash
-  SILK_CLIENT_ID=<client_id>
-  SILK_CLIENT_SECRET=<client_secret>
-  ```
-- Snyk credentials. Save these to `~/.secrets/snyk-creds.txt`:
-  ```bash
-  SNYK_API_TOKEN=<token>
-  ```
+- GitHub Personal Access Token.
+- Jira OAuth credentials.
+- Artifactory credentials.
+- Garasign credentials.
 
 Run the release script with the git tag created above as an argument and
 `--dry-run` to test for unexpected errors.
 
+```bash
+make_release_args=(
+    --jira-creds-file ~/.secrets/jira-creds.txt
+    --github-token-file ~/.secrets/github-token.txt
+)
+python ./etc/make_release.py "${make_release_args[@]:?}" --dry-run rX.Y.Z
 ```
-python ./etc/make_release.py \
-    --dry-run \
-    --jira-creds-file ~/.secrets/jira_creds.txt \
-    --github-token-file ~/.secrets/github_token.txt \
-    r1.2.3
-```
-
-If all goes well, run the command again without `--dry-run`, which should build
-and test the tarball and draft the GitHub release.
 
-### Troubleshooting make_release.py
+> [!TIP]
+> Export environment variables (e.g.`CMAKE_BUILD_PARALLEL_LEVEL`) to improve the speed of this command.
 
 If an error occurs, inspect logs the script produces, and troubleshoot as
 follows:
 
 - Use `--dry-run` to prevent unrecoverable effects.
 - If building the C driver fails, use an existing C driver build (ensure it is
-  the right version) with `--with-c-driver /path/to/cdriver/install`.
+  the right version) with `--with-c-driver /path/to/c-driver/install`.
 - Use `--skip-distcheck` to bypass time consuming checks when building the
   distribution tarball.
 - If the script succeeded at creating the distribution tarball, pass it directly
-  with `--dist-file ./build/mongo-cxx-driver-r1.2.3.tar.gz`.
+  with `--dist-file ./build/mongo-cxx-driver-rX.Y.Z.tar.gz`.
 
-## Push the tag
+If all goes well, run the command again without `--dry-run`. This should update Jira and create a draft release on GitHub.
 
-Review the build output and, assuming the distcheck target is successful, push
-the tag into the main remote:
+> [!TIP]
+> To avoid the Jira ticket tracking the release itself from preventing the script from completing successfully, pass `--allow-open-issues`. Carefully inspect the list of open tickets before using this flag!
```

Comment:
Consider removing tip. [This change](https://github.com/mongodb/mongo-cxx-driver/pull/1109/files#diff-61acca32013fbc56342f9c439770e4adff273cbfd3858a8e8fe62cf3cbfaf8eaR502) includes a check for the release ticket, but was not cherry-picked. If this PR is intended to be cherry-picked onto `releases/v3.10`, consider also including the check in the cherry-pick.

Reply:
This PR is meant for `master` only and will not be cherry-picked. Will remove the tip as suggested.